modules/bmo-attaching-a-non-bootable-iso-to-a-bare-metal-node.adoc

```diff
@@ -50,7 +50,7 @@ $ oc apply -f <node_name>-dataimage.yaml -n <node_namespace> <1>
 +
 [NOTE]
 ====
-You can physically reboot the node. You can also attach the `reboot.metal3.io` annotation or reset set the `online` status in the `BareMetalHost` resource. A forced reboot of the bare-metal node will change the state of the node to `NotReady` for awhile. For example, 5 minutes or more.
+To reboot the node, attach the `reboot.metal3.io` annotation, or reset set the `online` status in the `BareMetalHost` resource. A forced reboot of the bare-metal node will change the state of the node to `NotReady` for awhile. For example, 5 minutes or more.
```

Comment:
@gwynnemonahan  Just a very small nit. "awhile" is an adverb, as in "I'm going to read awhile." 

However, "a while" is the noun phrase, as in "I'm going to read for a while."